scss/core/_variables.scss

```diff
@@ -628,7 +628,7 @@ $input-btn-padding-y:         0.5625rem !default;
 $input-btn-padding-x:         1rem !default;
 $input-btn-font-family:       null !default;
 $input-btn-font-size:         1.125rem !default;
-$input-btn-line-height:       1.33334 !default;
+$input-btn-line-height:       1.3333 !default;
```

Comment:
Out of curiosity, why this change?

Reply:
Rounding up caused an off by 1 pixel alignment issue of the floating label and where the input text would be.